gen/aircraft.py

```diff
@@ -1357,13 +1370,32 @@ def configure_awacs(
     ) -> None:
         group.task = AWACS.name
         self._setup_group(group, AWACS, package, flight, dynamic_runways)
+        if not isinstance(flight.flight_plan, AwacsFlightPlan):
+            logging.error(f"Cannot configure AEW&C tasks for {flight} because it does not have an AEW&C flight plan.")
+            return
+
+        # Awacs task action
         self.configure_behavior(
             group,
             react_on_threat=OptReactOnThreat.Values.EvadeFire,
             roe=OptROE.Values.WeaponHold,
             restrict_jettison=True,
         )
 
+        group.points[0].tasks.append(AWACSTaskAction())
+        callsign = callsign_for_support_unit(group)
```

Comment:
This isn't correct, is it? The group's callsign isn't actually being set anywhere that I can see.

Reply:
You are correct. I just generated a callsign but forgot to assign it ^^